src/renderer/views/Playlist/Playlist.js

```diff
@@ -155,22 +164,32 @@ export default Vue.extend({
       })
     },
 
-    nextPage: function () {
-      const payload = {
-        query: this.query,
-        options: {
-          nextpageRef: this.nextPageRef
-        },
-        nextPage: true
+    getNextPage: function () {
+      switch (this.backendPreference) {
```

Comment:
Not sure if we should have this at the moment. If the show more button is only visible when local api is used then wouldn't this only error if someone has invidious as preferred backend and it had a fallback of local api for the playlist request?

Reply:
Changed it to check `this.infoData.infoSource` instead, that way it should work for the backend fallback too.